src/PrestaShopBundle/Form/Admin/Type/TranslateType.php

```diff
@@ -93,4 +138,32 @@ public function getBlockPrefix()
     {
         return 'translatefields';
     }
+
+    /**
+     * Get default locale.
+     *
+     * @param array $locales
+     *
+     * @return array
+     */
+    private function getDefaultLocale(array $locales)
```

Comment:
You can refactor this function:
```
private function getDefaultLocale(array $locales)
{
    $languageId = $this->defaultFormLanguageId ?
                $this->defaultFormLanguageId :
                $this->defaultShopLanguageId;
    
    foreach ($locales as $locale) {
        if ($locale['id_lang'] == $languageId) {
            return $locale;
        }
    }

    return reset($locales);
}
```

Reply:
:+1: thanks

Reply:
even shorter with short ternary: 
` $languageId = $this->defaultFormLanguageId ?: $this->defaultShopLanguageId;`

Reply:
fixed.